discovery-infra/test_infra/kubeapi_utils.py

```diff
@@ -91,3 +96,24 @@ def get_hostname_for_agent(agent, nodes_details):
             if interface['macAddress'].lower() == mac_address:
                 return node_metadata['name']
 
+
+def delete_kube_api_resources_for_namespace(kube_api_client, name, namespace):
+    # any resource that created within a KubeAPIContext will be deleted on
+    # context exit.
```

Comment:
replace with docstring

Reply:
why? this is not a description of the function just a side comment

Reply:
ok, so https://github.com/openshift/assisted-test-infra/pull/696#discussion_r611374577 is the important part